CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.1.3
+- Handled function arguments - dart keyword name collision
+- Fix travis tests. Auto create dynamic library if it doesn't exist.
```

Comment:
> Auto create dynamic library if it doesn't exist.

This you removed from the CL again?

Reply:
We don't create dynamic library in tests now, but manually call `pub run ffigen:setup` in travis before running tests

Reply:
Yes, which is why the changelog should be updated.

Reply:
Yeah, done